docs/bettercap-rdp-mitm.md

```diff
@@ -1,24 +1,60 @@
 # Run PyRDP alongisde Bettercap
 
-Here is a short tutorial on how to combine the MITM RPD tool and Bettercap to
-redirect all RDP sessions on a LAN segment. By default Bettercap poisons only
-the gateway so connections will need to cross the gateway for them to be
-poisoned.
+Here is a short tutorial on how to combine the PyRDP and Bettercap to
+redirect all RDP sessions on a LAN segment. By default, Bettercap's arp
+spoofing module will spoof the entire subnet over a given interface. We
+will spawn an instance of PyRDP for each incoming RDP connections if they
+are in our target list, otherwise we will forward them.
 
-Start bettercap in normal mode (you can use other options if you like).
+## Requirements
 
-    bettercap -I <interface>
+1. PyRDP  
+2. Our fork and branch of Bettercap [https://github.com/GoSecure/bettercap/tree/rdp-mitm]
+3. Our fork of Caplets [https://github.com/GoSecure/caplets/]
 
-Run the PyRDP mitm
+## Setup
 
-    python3 bin/pyrdp-mitm.py <dst-rdp-machine>
+Install PyRDP by following these steps : https://github.com/GoSecure/pyrdp#installing
 
-Add this iptables rule to redirect all traffic to the mitm
+Install Bettercap by running these commands :
 
-    iptables -t nat -A PREROUTING -p tcp --dport 3389 -j REDIRECT --to-ports 3389
+    sudo apt-get update
+    sudo apt-get install build-essential libpcap-dev libusb-1.0-0-dev libnetfilter-queue-dev
+    mkdir -p $GOPATH/src/github.com/bettercap/
+    cd $GOPATH/src/github.com/bettercap/
+    git clone https://github.com/GoSecure/bettercap.git -b rdp-mitm
+    cd bettercap
+    go get
+    go build
+    make build && sudo make install
 
-Have fun!
+Install Caplets by running these commands :
+
+    cd
+    git clone https://github.com/GoSecure/caplets.git
+    cd caplets
+    sudo make install
+
+## Usage
+
+Start bettercap with :
+
+    sudo bettercap -iface <interface> -caplet <caplet>
```

Comment:
Using sudo is incompatible with PyRDP installed in a `venv`. Sudo destroys the `PATH` and even `sudo -E` doesn't allow it to be overridden.

@Pourliver had a neat workaround:

```
-eval "set rdp.proxy.command $(which pyrdp-mitm.py);"
```